content/blog/scales-1-3-0/index.Rmd

```diff
@@ -0,0 +1,101 @@
+---
+output: hugodown::hugo_document
+
+slug: scales-1-3-0
+title: scales 1.3.0
+date: 2023-11-07
+author: Thomas Lin Pedersen
+description: >
+    scales 1.3.0 is a minor release focusing on streamlining the API and gradual improvements 
+    of the existing utilities
+
+photo:
+  url: https://unsplash.com/photos/gold-and-silver-round-frame-magnifying-glass-j06gLuKK0GM
+  author: Elena Mozhvilo
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [package] 
+tags: []
+---
+
+```{=html}
+<!--
+TODO:
+* [ ] Look over / edit the post's title in the yaml
```

Comment:
Needs updating

Reply:
As in, I need to tick off the list?

Reply:
Yup. That way we all know its been done.